livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterprereter.java

```diff
@@ -422,6 +423,8 @@ private String callRestAPI(String targetURL, String method, String jsonData)
         if (cause.getResponseBodyAsString().matches(SESSION_NOT_FOUND_PATTERN)) {
           throw new SessionNotFoundException(cause.getResponseBodyAsString());
         }
+        throw new LivyException(cause.getResponseBodyAsString() + "\n"
+            + ExceptionUtils.getFullStackTrace(ExceptionUtils.getRootCause(e)));
```

Comment:
The previous implementation use the original exception without the `ResponseBody`, the purpose of this PR is to add the ResponseBody to exception message, so that in frontend, the  error message is more meaningful in kerberos enviroment.